<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="player_health">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QLabel" name="bot_health">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startgame">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>360</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Начать игру</string>
    </property>
   </widget>
   <widget class="QComboBox" name="defend_zone">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Голова</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Тущовище</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ноги</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="attack_zone">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Голова</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Туловище</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ноги</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>136</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбери зону защиты</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбери зону атаки</string>
    </property>
   </widget>
   <widget class="QPushButton" name="attack_button">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Сделать ход</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitgame">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Выйти из игры</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Бот ударил</string>
    </property>
   </widget>
   <widget class="QLabel" name="bot_attack">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>260</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bot_defend">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>284</y>
      <width>52</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>110</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Бот защитил</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
